dvc.lock

```diff
@@ -2,6 +2,11 @@ schema: '2.0'
 stages:
   ingest:
     cmd: Rscript pipeline/00-ingest.R
+    deps:
+    - path: pipeline/00-ingest.R
+      hash: md5
+      md5: 29292ee2bef109914c423c9259aa8879
+      size: 22847
```

Comment:
**[Question, non-blocking]** I haven't seen us use this pattern before, what's the goal of making the script a dependency?

Reply:
It's used in both model repos now, with the thinking being that the pipeline should be restarted if the script changes. I'm not super tied to this setup though if you don't like it. I agree it's a bit weird.

Reply:
I think that makes sense, thanks!